spec/features/testing_infrastructure_spec.rb

```diff
@@ -1,6 +1,6 @@
-feature 'Testing infrastructure' do
-  scenario 'Can run app and check page content' do
-    visit('/')
-    expect(page).to have_content 'Testing infrastructure working!'
-  end
-end
+# feature 'Testing infrastructure' do
+#   scenario 'Can run app and check page content' do
+#     visit('/')
+#     expect(page).to have_content 'Testing infrastructure working!'
+#   end
+# end
```

Comment:
Would this test still work if uncommented?

Reply:
Obviusly not. This was just the first feature test to test if the infrastructure was actually working.